src/document/DocumentAnnotator.ts

```diff
@@ -97,10 +114,21 @@ export default class DocumentAnnotator extends BaseAnnotator {
             return;
         }
 
-        if (mode === Mode.HIGHLIGHT) {
-            this.annotatedEl.classList.add('ba-is-highlighting');
-        } else {
-            this.annotatedEl.classList.remove('ba-is-highlighting');
+        ['ba-is-highlighting', 'ba-is-create-region'].forEach(
+            className => this.annotatedEl && this.annotatedEl.classList.remove(className),
+        );
+
+        switch (mode) {
+            case Mode.HIGHLIGHT:
+                this.annotatedEl.classList.add('ba-is-highlighting');
+                this.annotatedEl.classList.remove('ba-is-create-region');
+                break;
+            case Mode.REGION:
+                this.annotatedEl.classList.add('ba-is-create-region');
```

Comment:
Can we standardize the naming of these classes? Maybe `ba-is-create--highlight` and `ba-is-create--region`?